anago

```diff
@@ -554,13 +557,20 @@ announce () {
   local announcement_text=/tmp/$PROG-announce.$$
 
   ((FLAGS_nomock)) || mailto=$USER@$DOMAIN_NAME
+  if [[ -n "${FLAGS_mailto}" ]]; then
+    mailto="${FLAGS_mailto}"
+  fi
 
   if [[ -n "$PARENT_BRANCH" ]]; then
     create_branch_announcement
   else
     create_announcement
   fi > $announcement_text
 
+  ((FLAGS_yes)) \
+   || common::askyorn -e "Pausing here. Confirm announce to $mailto" \
```

Comment:
Sorry, "yes" should be "ack". The point being, I'm all for session values being presented to the user, but in a less scattered way and all together in the "SESSION VALUES" section for a complete preview.  So --mailto or not, just present it in "SESSION VALUES".